taskcluster/host-build.sh

```diff
@@ -25,7 +25,11 @@ do_bazel_build
 
 do_deepspeech_binary_build
 
-do_deepspeech_python_build
+if [ "${runtime}" = "tflite" ]; then
+  do_deepspeech_python_build "--tflite"
```

Comment:
Pretty sure this is going to break platforms where the default runtime is already TFLite :/

Reply:
By breaking, I mean that people already using `deepspeech` dep on those platforms will get it broken.

Reply:
Those are `rpi3-build.sh` and `arm64-build.sh`, no?

Reply:
Yeah, I was too quick in commenting. You're right, we should indeed be safe and have this alternative `deepspeech-tflite` working as expected.

Reply:
Can I take this as an r+? :D

Reply:
yeah, sorry, i forgot